mage_ai/streaming/sinks/activemq.py

```diff
@@ -58,12 +58,16 @@ def batch_write(self, messages: List[Dict]):
         )
         for message in messages:
             if isinstance(message, dict):
-                data = message.get('data', message)
-                metadata = message.get('metadata', None)
+                data = message.get('message', message)
```

Comment:
The data format can't be changed: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/streaming/sinks/base.py#L47-L51